lib/elixir/lib/code.ex

```diff
@@ -802,8 +802,8 @@ defmodule Code do
   and errors.
 
   **Warning**: `string` can be any Elixir code and code can be executed with
```

Comment:
I think here it is correct to say, as it is said the `Code.eval_string/3`, that "**will** be executed with
  the same privileges as the Erlang VM"